proteinworkshop/datasets/pdb_dataset.py

```diff
@@ -130,21 +124,28 @@ class PDBDataModule(ProteinDataModule):
     def __init__(
         self,
         path: Optional[str] = None,
+        structure_dir: Optional[str] = None,
         pdb_dataset: Optional[PDBData] = None,
         transforms: Optional[Iterable[Callable]] = None,
         in_memory: bool = False,
         batch_size: int = 32,
         num_workers: int = 0,
         pin_memory: bool = False,
+        structure_format: str = "mmtf.gz",
         overwrite: bool = False,
     ):
         super().__init__()
         self.root = path
         self.dataset = pdb_dataset
         self.dataset.path = path
-        self.format = "mmtf.gz"
+        self.format = structure_format
         self.overwrite = overwrite
 
+        if structure_dir is not None:
+            self.structure_dir = pathlib.Path(structure_dir)
+        else:
```

Comment:
We may need to create these directories if they do not already exist.

Reply:
done @a-r-j